### PR TITLE
added check for area applied for

### PR DESCRIPTION
**Pull request by charliejade8888**

### CODE REVIEW

File: app/routes/action.js [hunk 1]
```diff
@@ -18,6 +18,22 @@ const getActionsForLandUses = (landUseCodes) => {
   })
 }
 
+const checkAreaAppliedForValid = (userSelectedActions, landParcel) => {
+  let message
+  for (const action of userSelectedActions) {
+    const areaAppliedFor = parseFloat(action.quantity)
+    const area = parseFloat(landParcel.area)
```
Comment (beclamide) [inline app/routes/action.js:25]: This is a constant within the for loop so should be hoisted out of it

File: app/routes/action.js [hunk 2]
```diff
@@ -18,6 +18,22 @@ const getActionsForLandUses = (landUseCodes) => {
   })
 }
 
+const checkAreaAppliedForValid = (userSelectedActions, landParcel) => {
+  let message
+  for (const action of userSelectedActions) {
```
[the remark this answers is not in the document]
Comment (beclamide) [reply app/routes/action.js:23]: would returning an array of messages cause issues elsewhere?

[the remark this answers is not in the document]
Comment (charliejade8888) [reply app/routes/action.js:23]: image
Actually, this does return an array 👍

File: app/routes/action.js (hunk 1 above)
Comment (charliejade8888) [reply app/routes/action.js]: Done 👍

**Review by beclamide — APPROVED** [review]

**Pull request merged**